src/Lib/Canvas.vala

```diff
@@ -98,6 +102,10 @@ public class Akira.Lib.Canvas : Goo.Canvas {
 
     private Goo.CanvasRect? hover_effect;
 
+    private bool ctrl_is_pressed = false;
+    private bool pan_mode_enabled = false;
+    private bool zoom_mode_enabled = false;
+    private bool pan_zoom_is_holding = false;
```

Comment:
Are these two variables and related methods used anywhere?
It's seems this is a leftover for the zoom handling, which I've already implemented in my WIP PR.
Please, remove variables and methods that are not used and are not part of the scope of this PR.

Reply:
Totally agree with you. I removed those variables and I took care of removing everything not related to the pan and click fixed rotation additions.